nodejs-addon-examples/test_tts_non_streaming_vits_zh_ll.js

```diff
@@ -13,7 +13,7 @@ function createOfflineTts() {
         dictDir: './sherpa-onnx-vits-zh-ll/dict',
       },
       debug: true,
-      numThreads: 1,
+      num_threads: 1,
```

Comment:
Could you change
https://github.com/k2-fsa/sherpa-onnx/blob/b4f1006864a7e86576be105d03541ba4eaaaf00e/scripts/node-addon-api/src/non-streaming-tts.cc#L47
to
```
 SHERPA_ONNX_ASSIGN_ATTR_INT32(num_threads, numThreads); 
```

---

<img width="887" alt="Screenshot 2024-05-29 at 10 30 21" src="https://github.com/k2-fsa/sherpa-onnx/assets/5284924/522e8eff-f39f-4483-b100-180acb44131d">